lib/models/asyncapi.js

```diff
@@ -205,21 +205,11 @@ class AsyncAPIDocument extends Base {
     return !!this._json[String(xParserCircle)];
   }
 
-  /**
-   * Callback used when crawling a schema.
-   * @callback TraverseSchemas
-   * @param {Schema} schema which is being crawled 
-   * @param {String} propName if the schema is from a property get the name of such
-   * @param {SchemaIteratorCallbackType} callbackType is the schema a new one or is the crawler finishing one.
-   * @returns {boolean} should the crawler continue crawling the schema?
-   * 
-   */
-
```

Comment:
I feel like we should require `Schema` instead of removing the entire JSDoc for the callback. What do you think? 🙂

Reply:
@jonaslagoni I agree. How would you suggest I approach it. (Sorry, fairly new to JSdoc)

Reply:
All you have to do is require the Schema model in the top of the model. The linter might not like a dependency that is never "used" per say, if it does just disable that check for that line.

Did that clarify how? :)

Reply:
@jonaslagoni I have tried to do it but no luck.
1) imported the schema.
`// eslint-disable-next-line no-unused-vars
const Schema = require('./schema');`
2) tried using `@requires Schema` tag.
any advice?

Reply:
1. step is correct 👌 
2. step is to just use the old JSDoc statements as is without modifying it. You dont have to use any tags 🙂

Reply:
But importing the schema makes no difference. Still getting the same error.

Reply:
@KhudaDad414 can you try committing the changes you have, better to spot the problem that way

Reply:
Just in case you did not run it, remember to re-generate the types with `npm run types` before trying to use the changes

Reply:
> Just in case you did not run it, remember to re-generate the types with npm run types before trying to use the changes

I run it.

Reply:
@jonaslagoni 
committed the changes.

Reply:
Ahh yea I see the problem 😅 2 sec.